hack/test-cmd.sh

```diff
@@ -45,6 +50,36 @@ function cleanup()
 trap "exit" INT TERM
 trap "cleanup" EXIT
 
+function exectest() {
+  testname=$1
+  testcmd=$2
+  echo "Running $testname..."
+
+  result=1
+  if [ -n "${VERBOSE-}" ]; then
```

Comment:
I think this is backwards: `-n "${VERBOSE-}"` means `$VERBOSE` is not empty, which means it should be verbose, right?